src/Moryx.Runtime.Endpoints/Databases/Endpoint/DatabaseController.cs

```diff
@@ -227,7 +227,7 @@ public async Task<ActionResult<DatabaseMigrationSummary>> MigrateDatabaseModel(s
 
         [HttpPost("{targetModel}/setup")]
         [Authorize(Policy = RuntimePermissions.DatabaseCanSetup)]
-        public ActionResult<InvocationResponse> ExecuteSetup(string targetModel, ExecuteSetupRequest request)
```

Comment:
Not 100% sure, is this a breaking change? I don't know how Asp translates this to http 🤔